src/Compiler/RefC/RefC.idr

```diff
@@ -452,9 +452,6 @@ const2Integer c i =
         _ => i
 
 
-
```

Comment:
Although I can see the temptation, I am not sure if we should be making arbitrary whitespace changes given that we don't have community or compiler-specific guidelines on the subject and the author might well have made intentional choices around these newlines based on their personal preferences.

Reply:
Fair enough.  I can revert those vertical whitespace changes if needed.  I agree I made it look more pleasing according to my personal preferences. We do need community guidelines.

Reply:
Personally, I would say these newline changes should be reverted before merging, but if you would like to keep them and another maintainer decides this is ready to merge with these changes in place, I won't put up a fuss over such a minor detail. I just consider it a courtesy to the original author of the file.

Reply:
Ok, I have reverted these whitespace changes.